e2e/portalicious/tests/SendMessage/SendTemplatedMessage.spec.ts

```diff
@@ -0,0 +1,55 @@
+import { test } from '@playwright/test';
+
+import { SeedScript } from '@121-service/src/scripts/seed-script.enum';
+import { seedIncludedRegistrations } from '@121-service/test/helpers/registration.helper';
+import { resetDB } from '@121-service/test/helpers/utility.helper';
+import { getAccessToken } from '@121-service/test/helpers/utility.helper';
+import { registrationsPV } from '@121-service/test/registrations/pagination/pagination-data';
+
+import BasePage from '@121-e2e/portalicious/pages/BasePage';
+import LoginPage from '@121-e2e/portalicious/pages/LoginPage';
+import RegistrationsPage from '@121-e2e/portalicious/pages/RegistrationsPage';
+
+const sendingMessageToast =
+  'Closing this notification will not cancel message sending.';
+
+test.beforeEach(async ({ page }) => {
+  await resetDB(SeedScript.nlrcMultiple);
+  const programIdPV = 2;
+  const pvProgramId = programIdPV;
+
+  const accessToken = await getAccessToken();
+  await seedIncludedRegistrations(registrationsPV, pvProgramId, accessToken);
+
+  // Login
+  const loginPage = new LoginPage(page);
+  await page.goto('/');
+  await loginPage.login(
+    process.env.USERCONFIG_121_SERVICE_EMAIL_ADMIN,
+    process.env.USERCONFIG_121_SERVICE_PASSWORD_ADMIN,
+  );
+});
+
+test('[31076] Send templated message', async ({ page }) => {
+  const basePage = new BasePage(page);
+  const registrations = new RegistrationsPage(page);
+
+  const projectTitle = 'NLRC Direct Digital Aid Program (PV)';
+
+  await test.step('Select program', async () => {
+    await basePage.selectProgram(projectTitle);
+  });
+
+  await test.step('Send templated message', async () => {
+    await registrations.selectAllPAs();
+    await registrations.selectBulkAction('Message');
+    await registrations.selectTemplatedMessage('Include');
+    await registrations.clickContinueToPreview();
+    await registrations.validateMessagePresent(
+      'This is a message from the Red Cross.',
+    );
+    await registrations.sendMessage();
+
+    await registrations.validateToastMessage(sendingMessageToast);
```

Comment:
Same as the other test - validate in activity log if possible.

Reply:
Done!